hokusai/commands/config.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from collections import OrderedDict
+from subprocess import check_output, CalledProcessError, STDOUT
+import yaml
+
+from hokusai.config import HokusaiConfig
+from hokusai.common import print_red, print_green
+
+def pull_config(context):
```

Comment:
Inline comments describing the command and args would be useful to devs looking at the code. Though obviously the comments are all compiled in `bin/hokusai` to be accessed by users of the tool, inline would be good for contributing devs.